### PR TITLE
[MIRROR] Lets all heads set alert level, restores "Station Time" option to status displays

### PR DESCRIPTION
**Pull request by CHOMPStationBot**
Original PR: https://github.com/VOREStation/VOREStation/pull/9213
--------------------
*Moves alert level options to the command staff section and lets command staff use them.  This matches Polaris behavior and pre-TGUI behavior.  This fixes https://github.com/VOREStation/VOREStation/issues/9212 .
*Renames "Site Director-Only Actions" to "Site Manager-Only Actions" for consistency with the name used for that role in other places.
*Adds the "Station Time" option for status displays from the pre-TGUI communications console to the TGUI communications console.  Tested, consistent with Polaris and pre-TGUI behavior.
*Renames the "Shuttle ETA" option for status displays to "Tram ETA", because we have a tram and not a shuttle.

### CODE REVIEW

**Comment by Razgriz1032** [comment]
This needs a TGUI bundle regeneration, or something.

**Comment by Shadow-Quill** [comment]
This fixes #707

**Comment by Darlantanis** [comment]
I dunno if this is can be merged or not (Raz's comment above) but obviously we're not going to change the Shuttle ETA message because we have a shuttle, not a tram.

**Comment by cadyn** [comment]
> I dunno if this is can be merged or not (Raz's comment above) but obviously we're not going to change the Shuttle ETA message because we have a shuttle, not a tram.

Changed it back to shuttle ETA

File: code/modules/tgui/modules/communications.dm [hunk 1]
```diff
@@ -104,9 +104,10 @@
 		"line_2" = (stat_msg2 ? stat_msg2 : "-----"),
 
 		"presets" = list(
-			list("name" = "blank",    "label" = "Clear",       "desc" = "Blank slate"),
-			list("name" = "shuttle",  "label" = "Shuttle ETA", "desc" = "Display how much time is left."),
-			list("name" = "message",  "label" = "Message",     "desc" = "A custom message.")
+			list("name" = "blank",    "label" = "Clear",        "desc" = "Blank slate."),
+			list("name" = "time",     "label" = "Station Time", "desc" = "The current time according to the station's clock."),
+			list("name" = "shuttle",  "label" = "Shuttle ETA",     "desc" = "Display how much time is left."),  //CHOMPEdit we have a shuttle not a tram silly virgo//VOREStation Edit - Shuttle ETA -> Tram ETA because we use trams
```
Comment (Shadow-Quill) [inline code/modules/tgui/modules/communications.dm:109]: Do we need to keep the original Virgo comment?

Comment (cadyn) [reply code/modules/tgui/modules/communications.dm:109]: Yea, it's there to remind us why virgo changed it in the first place and why we needed to change it back.

**Comment by Shadow-Quill** [comment]
@Darlantanis No travis conflicts. Is this okay now?

**Comment by Razgriz1032** [comment]
I appreciate the updated TGUI file.

**Pull request merged**